dvc/logger.py

```diff
@@ -170,15 +175,25 @@ def setup(level=logging.INFO):
     logging.config.dictConfig(
         {
             "version": 1,
-            "filters": {"exclude_errors": {"()": ExcludeErrorsFilter}},
+            "filters": {
+                "exclude_errors": {"()": ExcludeErrorsFilter},
+                "exclude_info": {"()": ExcludeInfoFilter},
+            },
             "formatters": {"color": {"()": ColorFormatter}},
             "handlers": {
-                "console": {
+                "console_info": {
+                    "class": "dvc.logger.LoggerHandler",
+                    "level": "INFO",
+                    # "formatter": "color",
```

Comment:
?

Reply:
see below explanation

Reply:
Yeah, but why do we have a commented-out line here?

Reply:
ah yes, it can be removed as it's unneeded. on the other hand the colour formatter seems to pass through `INFO` without altering so might just uncomment for future-proofing.